src/Symfony/Component/Uid/Uuid.php

```diff
@@ -100,27 +95,23 @@ public function getType(): int
         return uuid_type($this->uuid);
     }
 
-    public function getVariant(): int
```

Comment:
Nobody uses variants, isn't it? Let's drop it to make our API simpler.
`uuid_variant()` still exists for ppl that need it.